packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -325,6 +378,9 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
 Future<T> showModalBottomSheet<T>({
   @required BuildContext context,
   @required WidgetBuilder builder,
+  Color color,
```

Comment:
It's not crazy to consider passing in a `BottomSheetThemeData bottomSheetTheme` param to this function. I'm not sure if there's a precedence for this, and it might not make total sense here, just something to consider.

Reply:
This is something I considered but landed on not doing, it doesn't seem like there's a precedence for it and the BottomSheet is simple enough that we won't be having to add more visual properties later on.